### PR TITLE
TASK: Adjust version numbers in tooling configuration

### PR DESCRIPTION
**Pull request by bwaidelich**
Updates for the 7.0 branch:

- conf.py of neos/neos and neos/media
- .travis.yml
- apigen.yml

### CODE REVIEW

File: Neos.Media/Documentation/conf.py [hunk 1]
```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'dev-master'
+version = 'dev-7.0'
```
Comment (bwaidelich) [inline Neos.Media/Documentation/conf.py:62]: @albe @kdambekalns I think now I completely mixed it up o.O

**Comment by kdambekalns** [comment]
"Short" should be `7.0` and "long" should be `7.0.x`, when I think about it "from scratch".

File: Neos.Media/Documentation/conf.py (hunk 1 above)
Comment (bwaidelich) [inline Neos.Media/Documentation/conf.py:62]:
```suggestion
version = '7.0'
```

File: Neos.Media/Documentation/conf.py [hunk 2]
```diff
@@ -59,9 +59,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = 'dev-master'
+release = 'dev-7.0.x'
```
Comment (bwaidelich) [inline Neos.Media/Documentation/conf.py:64]:
```suggestion
release = '7.0.x'
```

File: Neos.Neos/Documentation/conf.py [hunk 3]
```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'dev-master'
+version = 'dev-7.0'
```
Comment (bwaidelich) [inline Neos.Neos/Documentation/conf.py:62]:
```suggestion
version = '7.0'
```

File: Neos.Neos/Documentation/conf.py [hunk 4]
```diff
@@ -59,9 +59,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = 'dev-master'
+release = 'dev-7.0.x'
```
Comment (bwaidelich) [inline Neos.Neos/Documentation/conf.py:64]:
```suggestion
release = '7.0.x'
```

**Review by bwaidelich — COMMENTED** [review]
Like this, @kdambekalns ?

**Review by kdambekalns — APPROVED** [review]

**Review by albe — APPROVED** [review]
Needs #3232 to make tests work but this looks fine and should not affect tests at all.

**Pull request merged**